compose.yaml

```diff
@@ -5,6 +5,6 @@ services:
       environment:
         - POSTGRES_DB=${POSTGRES_DB}
         - POSTGRES_PASSWORD=${POSTGRES_PASSWORD}
-        - POSTGRES_USER$=${POSTGRES_PASSWORD}
+        - POSTGRES_USER=${POSTGRES_USER}
```

Comment:
Password can probably be passed as a docker secret. Maybe it will be an overall better approach.
Check [this doc](https://docs.docker.com/compose/how-tos/use-secrets/) at some point

Reply:
I'm using .env file for a reason: there are several places where I need these variables. Maybe it's a design issue, but now I don't have enough experience with Java to judge.